install.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-set -xuo pipefail
+set -uo pipefail
```

Comment:
What's the idea here?

Reply:
The idea is that install.sh (and upgrade.sh) are now mature enough to suppress -x  verbose tracing.
Some users do execute the scripts by hand.